subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -545,6 +545,17 @@ include::sample[dir="snippets/configurationCache/testKit/kotlin",files="src/test
 
 If problems with the configuration cache are found then Gradle will fail the build reporting the problems, and the test will fail.
 
+[TIP]
+====
+A good testing strategy for a Gradle plugin is to run its whole test suite with the configuration cache enabled.
```

Comment:
Can you make it explicit in the note _what_ you need to write to enable configuration cache?

Reply:
It's already demonstrated with a snippet right above that note.
Do you still think it should also be repeated here?

Reply:
Ok I missed that it was the case. Should be fine then.